### PR TITLE
revert cassandra sweep changes

### PR DESCRIPTION
**Pull request by nziebart**
**Goals (and why)**:
Revert cassandra sweep changes, until we have a solution for not pulling values into memory

**Implementation Description (bullets)**:

**Concerns (what feedback would you like?)**:

**Where should we start reviewing?**:

**Priority (whenever / two weeks / yesterday)**:

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/2521)

### CODE REVIEW

**Comment by tpetracca** [comment]
is it not possible to do a git revert here?

**Comment by nziebart** [comment]
no, this was a feature branch and the desired revert was only a portion of the branch

**Comment by nziebart** [comment]
ultimately what we did was individually revert all of the cassandra-related files from the squashed commit

File: atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java [hunk 1]
```diff
@@ -38,4 +44,48 @@ protected KeyValueService createKeyValueService() {
...
+        assertThat(getAllCandidates(conservativeRequest(PtBytes.EMPTY_BYTE_ARRAY, 40L, 1)))
+                .containsExactly(ImmutableCandidateCellForSweeping.builder()
+                        .cell(cell(1, 1))
+                        .sortedTimestamps(new long[] { 10L, 20L })
+                        .isLatestValueEmpty(false)
+                        .numCellsTsPairsExamined(2)
+                        .build());
+    }
+
+    // TODO(nziebart): check that this is ok to fail, and delete
```
Comment (hsaraogi) [inline atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java]: Looks like its OK to fail now that `minUncommitedTimestamp` is no longer a thing when getting the candidate cells. https://github.com/palantir/atlasdb/blob/develop/atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepableCellFilter.java#L95-L101 will ensure that the latest commited value is not swept.

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/paging/RowGetter.java [hunk 2]
```diff
@@ -94,4 +94,4 @@ private SlicePredicate getSlicePredicate(ColumnFetchMode fetchMode) {
         predicate.setSlice_range(slice);
         return predicate;
     }
-}
+}
```
Comment (hsaraogi) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/paging/RowGetter.java]: New line, or the file appears in the diff `git diff 0.60.1..HEAD --name-only | grep cassandra`.

File: atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java [hunk 3]
```diff
@@ -69,23 +68,4 @@ public void returnCandidateIfTwoCommittedTimestamps() {
...
-    // TODO(nziebart): check that this is ok to fail, and delete
-    @Ignore
-    @Test
-    public void doNotReturnCandidateWithCommitedEmptyValueIfConservative() {
-        new TestDataBuilder().putEmpty(1, 1, 10L).store();
-        assertThat(getAllCandidates(conservativeRequest(PtBytes.EMPTY_BYTE_ARRAY, 40L, 1))).isEmpty();
-    }
-
-    @Test
-    public void returnCandidateWithCommitedEmptyValueIfThorough() {
```
Comment (hsaraogi) [inline atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java]: Why are we deleting this one?

File: atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java (hunk 1 above)
Comment (nziebart) [reply atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java]: yep, @fsamuel-bs looked at this with the same conclusion. filtering it in the shim is just an optimization

File: atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java (hunk 3 above)
Comment (nziebart) [reply atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java]: the concept of whether or not to empty values was based on an optimization in the shim which no longer exists. so this test doesn't really make sense anymore

Comment (hsaraogi) [reply atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java]: cool.

**Pull request merged**